src/pages/components/WebView.qml

```diff
@@ -38,9 +38,18 @@ WebContainer {
         contentItem.sendAsyncMessage(name, data)
     }
 
+    function getThumbnailSize() {
+        var largeScreen = (Screen.sizeCategory > Screen.Medium)
+        var width = Screen.width - (largeScreen ? (2 * Theme.horizontalPageMargin) : 0)
+        var height = largeScreen ? Theme.itemSizeExtraLarge + (2 * Theme.paddingLarge) : Screen.height / 5
+
+        return Qt.size(width, height)
```

Comment:
Design feedback: original capture could be like width of the device and height calculated with the above aspect ratio. Then in TabItem we could scale down the image to the size of the tab item.